src/main/java/com/redhat/devtools/intellij/lsp4mp4ij/psi/internal/graphql/TypeSystemDirectiveLocation.java

```diff
@@ -0,0 +1,15 @@
+package com.redhat.devtools.intellij.lsp4mp4ij.psi.internal.graphql;
+
+public enum TypeSystemDirectiveLocation {
```

Comment:
Add Javadoc and if it is possible add a link to the doc which defines those constants.

Reply:
done